gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilPlugin.kt

```diff
@@ -427,6 +436,7 @@ internal class Variant private constructor(
   val project: Project,
   val compileTaskProvider: TaskProvider<KotlinCompile>,
   val androidVariant: BaseVariant?,
+  val androidSourceSets: List<AndroidSourceSet>?,
```

Comment:
Is null treated differently than an empty list?

Reply:
I really want to represent absence here like with the variant above. I feel better this way.